tls/s2n_server_extensions.c

```diff
@@ -72,12 +74,21 @@ int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
         total_size += 4;
     }
 
+    if (conn->actual_protocol_version >= S2N_TLS13) {
+        total_size += 6;
```

Comment:
we should be moving inlined hardcoded sizes here to functions returning sizes in extensions eg. 
`s2n_extensions_server_supported_versions_send_size(conn)`

Reply:
will do! should i address the other hardcoded sizes in this PR (ex line 74 shown above)?

Reply:
it'd be nice but I don't think it's necessary until we start splitting all extensions into their own files and into a unified interface